libraries/standard/mqtt/src/mqtt.c

```diff
@@ -23,14 +23,23 @@
 
 #include "mqtt.h"
 
-/*-----------------------------------------------------------*/
-
-static int32_t sendPacket( MQTTContext_t * pContext, size_t bytesToSend )
+static int32_t _sendPacket( MQTTContext_t * pContext,
```

Comment:
I think just linking this is easier than explaining why _ should not be used.
https://community.sonarsource.com/t/misra-rule-20-1-check-is-not-accurate/3860

Reply:
We were following the naming conventions listed here (https://docs.aws.amazon.com/freertos/latest/lib-ref/c-sdk/main/guide_developer_styleguide.html). I think we will have to re evaluate the guidelines based on this explanation.